### PR TITLE
Initial import of expert expectations

### PR DESCRIPTION
**Pull request by QuintinWillison**
Includes content from [this internal Wiki page](https://ably.atlassian.net/wiki/spaces/DEL/pages/895090726/Managing+our+freelancers).

You can see _a_ rendered version of this document [here](https://github.com/ably/ably.github.io/blob/expert-expectations/docs/experts/expectations.md), however I'm yet to work out a system for this (still incubating) repository for previewing the GitHub Pages rendering for pull requests (i.e. in a Jekyll context). I've also got work to do on the Jekyll theme / layout (everything is default, out-of-the-box GitHub Pages right now).

This particular document has been identified as an immediate priority for publication, so while I acknowledge that there is yet to be an internal decision record defined to validate the creation of the GitHub pages site at ably.github.io and/or whether it ultimiately gets `CNAME`'d or not (see [this internal Slack thread](https://ably-real-time.slack.com/archives/CURL4U2FP/p1635939554012600)), I would appreciate it if reviewers could focus on the content - regardless of whether it ends up in a different home eventually.

**TL;DR:** Reviewers, please focus on the _what_, not the _how_.

### CODE REVIEW

**Review by niksilver — CHANGES_REQUESTED** [review]
Nice. Some changes I'd like to see...
* Put the definitions at the top. Then the reader understands them when they're first used.
* "Commit to the minimum number of regular hours you have agreed with us" - Should be the first item on the list. And should probably be accompanied by a comment saying the (almost) obvious, which is something like "and make sure we know what that commitment is".
* "In the same way... from you." seems like a long and tortuous sentence! Here's my attempt at simplifying it: "When things aren't going so well for you in meeting your responsibilities we will do our best to be flexible and understanding. We also expect the same from you."

Aren't we also supposed to have an in-person stand-up every week? I thought that's been very effective. Should be mentioned.

**Comment by QuintinWillison** [comment]
Thanks, @niksilver - I think I've responded to all of your points by way of new commits.

The only one I wasn't sure about was the "(almost) obvious" point relating to commitment. This, perhaps, crosses over into the domain of what we've contractually agreed with individuals. So, for the time being, I think it's ok to omit that for this first-pass publish of this document. We can always iterate later.

File: docs/experts/expectations.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+At the start of each work session, publish a self-stand-up via Slack, to include:
+
+- What you achieved last session
+- What Jira ticket you will be working on this session
+
+Attend the regular, in-person stand-up calls, as agreed with us.
+These are typically held weekly.
+
+You must flag via Slack when you are running low on work.
+We'll then work with you to line up the next series of Jira tickets.
```
Comment (AndyNicks) [inline docs/experts/expectations.md:24]: Jira or Github ticket?

Comment (QuintinWillison) [reply docs/experts/expectations.md:24]: I took this from @jamienewcomb's [original document](https://ably.atlassian.net/wiki/spaces/DEL/pages/895090726/Managing+our+freelancers) (internal Confluence page).

I was comfortable leaving this as just "Jira" here:

- This page refers to "GitHub First" elsewhere, and I intend to _also_ publish (move) our [GitHub First](https://ably.atlassian.net/wiki/spaces/SDK/pages/1489174532/GitHub+First) (internal Confluence page) policy to this site soon so that is also open / public.
- From a delivery perspective, which much of this content relates to, Jira is still our primary tool for queuing work in priority order as well as tracking progress.

**Review by AndyNicks — APPROVED** [review]

**Review by jamienewcomb — APPROVED** [review]
looks good to me. You added the thing about stand ups so thank you :)

**Review by jaley — APPROVED** [review]
Looks good, thanks!

**Review by niksilver — APPROVED** [review]

**Comment by niksilver** [comment]
Happy to approve.
